examples/plotting/plot_rotated_rectangle.py

```diff
@@ -0,0 +1,64 @@
+"""
+====================================
+Drawing a rotated rectangle on a map
+====================================
+
+This example will demonstrate how to draw a rectangle that is rotated relative
+to the axes on a map using :meth:`~sunpy.map.GenericMap.draw_quadrangle`.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord, SkyOffsetFrame
+
+import sunpy.data.sample
+import sunpy.map
+
+################################################################################
+# Let's start with a sample image of AIA 171.
+
+# sphinx_gallery_defer_figures
+
+aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+fig = plt.figure()
+ax = fig.add_subplot(projection=aia_map)
+aia_map.plot(axes=ax, clip_interval=(1, 99.99) * u.percent)
```

Comment:
I suggest moving this plotting code to the last code block so that you don't need to keep deferring the figure output.  Unlike the example that used as a template, there's no value in this example to have the plotting code early.